src/core/Hocon/Akka.Hocon.csproj

```diff
@@ -1,10 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.4;net461</TargetFrameworks>
+    <TargetFrameworks>net45;netstandard1.4</TargetFrameworks>
```

Comment:
please change this line to target both net45 and netstandard1.6 to reflect what is in the main repo.